README.md

```diff
@@ -0,0 +1,43 @@
+### FortiSOAR Connector Engine
+
+This package will support in local development of FortiSOAR connectors. This package include default import for 
```

Comment:
This library bundles multiple utilities to assist FortiSOAR integrations developments

Reply:
Done